SdkExample/Controllers/ActionsViewController.h

```diff
@@ -2,8 +2,8 @@
 //  ActionsViewController.h
 //  SdkExample
 //
-//  Created by Julian Oes on 31/08/16.
-//  Copyright © 2016 Julian Oes. All rights reserved.
+//  Created by Steven Hall on 9/13/17.
+//  Copyright © 2017 yuneec. All rights reserved.
```

Comment:
😞

Reply:
Na no worries 😄

Reply:
I let Xcode clean up some of the code, we can certainly put your name back in there if you'd like haha